app/categories/routes.py

```diff
@@ -65,7 +65,8 @@ def terrestres(ext):
         if road_pro_displacement:
             autofill_form(form_pro, road_pro_displacement, request)
         return render_template(
-            'terrestres.html', form_commute=form_commute, form_pro=form_pro)
+            'terrestres.html',
+            form_commute=form_commute, form_pro=form_pro, segment='terrestres')
```

Comment:
@MartinHerr Why did you use `terrestres` in the segment here? Don't we want to be more general?
```suggestion
            form_commute=form_commute, form_pro=form_pro, segment='terrestres')
```